cmd/deploy.go

```diff
@@ -12,13 +12,23 @@ import (
 )
 
 var (
+	deployExample = `
+Deployment you would like to deploy to Airflow cluster:
+
+  $ astro deploy <deployment name>
+
+Menu will be presented if you do not specify a deployment name:
```

Comment:
@schnie something like this?

Reply:
or `Menu with list of available deployment names will be presented if you do not specify a deployment name:`

Reply:
This works!